/**:
  ros__parameters:
    interface_mod: "CAN"                              # CAN or SERIAL
    base_frame_id: "base_link"
    command_timeout_ms: 1000.0
    data_publish_rate: 100.0
    reverse_gear_enabled: true
    gear_shift_velocity_threshold: 0.1
    max_steering_wheel_angle: 480.0
    min_steering_wheel_angle: -480.0
    max_steering_wheel_angle_rate: 300.0
    check_steering_angle_rate: false
    enable_emergency: false
    emergency_stop_acceleration: -5.0
    soft_stop_acceleration: -2.5
    add_emergency_acceleration_per_second: -1.5
    enable_cmd_timeout_emergency: false
    enable_long_actuation_mode: true
    vcu_mode: "normal"                      # normal, debug_steer, debug_pedal